cypress/integration/online-access.ts

```diff
@@ -0,0 +1,58 @@
+describe("Online workspaces access", () => {
```

Comment:
Can you add comments for the main steps of the test ? Makes them more legible and maintainable

See cypress/integration/classification.ts

Reply:
Done in 71c9ccfcc8aae75ec8100476df54a944132cece1